src/components/TestimonialSlider.jsx

```diff
@@ -47,48 +64,48 @@ const TestimonialSlider = () => {
   }, []);
 
   return (
-    <Grid container style={styles.testimonialContainer}>
-      {testimonials.length !== 0 && (
-        <>
-          <Grid item xs={1} style={styles.sliderItem}>
-            <IconButton
-              onClick={() => {
-                sliderHandler(-1);
-                stopTimer();
-              }}
-            >
-              <ChevronLeftIcon style={styles.sliderButton} fontSize="large" />
-            </IconButton>
-          </Grid>
-          <Grid
-            item
-            xs={10}
-            style={{ ...styles.sliderItem, ...styles.sliderCardContainer }}
-          >
-            <Testimonial slider={slider} data={testimonials[slider - 1]} />
-          </Grid>
-          <Grid item xs={1} style={styles.sliderItem}>
-            <IconButton
-              onClick={() => {
-                sliderHandler(1);
-                stopTimer();
-              }}
+    <Box>
```

Comment:
what does this Box do?

Reply:
it gives some extra margin around the component. 
moved the class from the grid to that one so it is seems more clear 👍